### PR TITLE
"Unknown" Language

### LINKED ISSUE #783: Need for "Unknown" languages code(s)
We recently saw a release with a made up language as base spoken language and subs over the entire feature, in this case english subs. `EN-XX`, `XX-EN`, and `EN-EN` all don't apply in this case. We had confirmation from the production company this is a made up language, like GoT languages. 

Doing some further research we need:
- Code for “Unknown” or “Made up” Language – think GoT or SW weird whatever alien language
- Code for Language we know is real but likely marked as a “dead” language that will never get a registered code via IANA.

### PR DESCRIPTION
**Pull request by SteveLLamb**
Update for unknown lang, closes #783 

This is actually used in CLDR now https://unicode.org/cldr/charts/44/summary/root.html#59146e1e75957ebd

### CODE REVIEW

**Comment by palemieux** [comment]
Can we also update the website to emphasize that `und` should only be used when a tag is required?

**Comment by SteveLLamb** [comment]
> Can we also update the website to emphasize that `und` should only be used when a tag is required?

Would it work for you to put that in the notes area for this code? Do you have any specific language you want to use?

**Comment by palemieux** [comment]
> Would it work for you to put that in the notes area for this code?

I think the warning to use `und` only when the absence of a language tag does not mean "unknown" should go here: https://registry-page.isdcf.com/languages/

For example, `und` can be used in the `language` element of a Sound Track File asset in the CPL since the absence of the `language` element does not mean unknown language, but no language.

Makes sense?

**Comment by SteveLLamb** [comment]
@palemieux this is addressed in https://github.com/ISDCF/registries-site/pull/80

File: src/main/data/languages.json [hunk 1]
```diff
@@ -2975,6 +2975,15 @@
...
   },
+  {
+    "dcncLanguage": "Unknown language",
+    "dcncTag": "UND",
+    "rfc5646Tag": "und",
+    "use": [
+      "audio",
+      "text"
+    ]
+  },
```
Comment (palemieux) [inline src/main/data/languages.json:2986]:
```
    "comments": [
      "Only to be used in fields or elements where the absence of value does not mean 'Unknown language'."
    ],
```

**Review by palemieux — CHANGES_REQUESTED** [review]

File: src/main/data/languages.json (hunk 1 above)
Comment (SteveLLamb) [reply src/main/data/languages.json:2989]: addressed in https://github.com/ISDCF/registries/pull/830/commits/845697fa805ea0d8ae721efa49ad0a2fb7783f4b

**Pull request merged**